src/citrine/resources/project.py

```diff
@@ -157,6 +157,22 @@ def share(self,
             "resource": {"type": resource_type, "id": resource_id}
         })
 
+    def make_public(self,
+                    resource_type: str,
+                    resource_id: str) -> Dict[str, str]:
```

Comment:
Can we return something more instructive than `Dict[str, str]` for a user-facing method?  Or minimally: document what the user should expect if the call is successful vs not successful.

Reply:
We should be making full docstrings (description, parameters, returns, and anything else as appropriate) for every method.